ptp/c4u/clock/clock.go

```diff
@@ -27,20 +31,26 @@ const (
 	ClockClassUncalibrated ptp.ClockClass = ptp.ClockClass52
 )
 
+type DataPoint struct {
+	PHCOffset            time.Duration
+	OscillatorOffset     time.Duration
+	OscillatorClockClass ptp.ClockClass
```

Comment:
Wouldn't it be easier to store oscillatorState structure here? :)

Reply:
I wanted to keep the `oscillatorState` struct hidden, it's not even exported by the package. Who knows how we end up getting this data in the future.